tests/utils.py

```diff
@@ -109,3 +111,7 @@ def switch_to_ta_grading(project_api_mock, review_group_id=1):
         UserAwareXBlockMixin.TA_REVIEW_KEY: review_group_id
     }
     project_api_mock.get_user_roles_for_course.return_value = [{'role': role} for role in ALLOWED_OUTSIDER_ROLES]
+
+
+def make_workgroup(workgroup_id, users):
```

Comment:
I'd suggest deleting this function. Its only purpose is to make the code more compact (and less clear)